src/hosted.js

```diff
@@ -131,12 +133,15 @@
     }
 
     // Support loading from src/ or dist/ to make local dev easier
-    require(["bramble/client/main"], function(Bramble) {
+    if(window.location.pathname === "/dist/hosted.html") {
```

Comment:
OK, I'll rework.

Speaking of this, can you tell me how we do paths for staging/prod so I can also fix my URL scheme for cached URLS in #549?